pkg/kubelet/eviction/helpers.go

```diff
@@ -203,7 +203,8 @@ func parseThresholdStatement(signal evictionapi.Signal, val string) (*evictionap
 		if percentage < 0 {
 			return nil, fmt.Errorf("eviction percentage threshold %v must be >= 0%%: %s", signal, val)
 		}
-		if percentage > 100 {
+		// percentage is a float and should not be greater than 1(100%)
```

Comment:
I'd add a space here @shawnhanx.
```suggestion
		// percentage is a float and should not be greater than 1 (100%)
```